roles/xmpp/tasks/prosody.yml

```diff
@@ -37,6 +34,23 @@
   template: src=prosody.cfg.lua.j2 dest=/etc/prosody/prosody.cfg.lua group=root owner=root
   notify: restart prosody
 
+- name: Copy SSL private key and cert
```

Comment:
OK, now I understand. Changing the group ownership to ssl-cert seems like a good idea though, doesn't it?
There's already an open issue for it at https://github.com/letsencrypt/letsencrypt/issues/1425